core/state_transition_ovm.go

```diff
@@ -0,0 +1,196 @@
+package core
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+var GodAddress = common.HexToAddress("0x444400000000000000000000000000000000000")
```

Comment:
This is a temporary value -- was a bit confused about our actual plans for this address.

Reply:
The god address is pulled in through this CLI flag - https://github.com/ethereum-optimism/go-ethereum/blob/b2e6790696a97e5e7c7c0c7f6facd4b66478d51f/cmd/utils/flags.go#L789

The plan is to remove it, but then we need a code path in the transaction ingestion pipeline that skips signature verification, since L1 did the signature verification. We talked about not incrementing any nonces for these kinds of txs. Would it make sense to get the L1 tx and compute the `from` and increment that nonce?